lab-lee/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "lab-lee",
+  "version": "1.0.0",
+  "description": "**About This Project**",
+  "main": "gulpfile.js",
+  "dependencies": {},
+  "devDependencies": {
+    "eslint": "^3.5.0",
+    "gulp": "^3.9.1",
+    "gulp-eslint": "^3.0.1",
+    "gulp-mocha": "^3.0.1",
+    "mocha": "^3.0.2",
+    "mocha-eslint": "^3.0.1",
+    "uuid": "^2.0.3"
```

Comment:
uuid is actually a regular dependency, as your program breaks if you try to run it without it. Your main functionality depends on creating an instance of Client, which uses the uuid library to generate a random number. If you try to run your program and don't have uuid installed, it errors out.